src/compas/data/encoders.py

```diff
@@ -78,7 +79,7 @@ class DataEncoder(json.JSONEncoder):
     >>> from compas.data import json_dump
     >>> from compas.geometry import Point
     >>> point = Point(0, 0, 0)
-    >>> json_dump(point, 'point.json')
+    >>> json_dump(point, compas.get('point.json'))
```

Comment:
since tests are run in shuffled order, the load test below can run before the dump test above, so, I created the `point.json` file in the `data` folder and the `compas.get` just makes sure it's getting that one. the import is right above